basyx/aas/adapter/http.py

```diff
@@ -23,7 +23,7 @@
 import werkzeug.routing
 import werkzeug.urls
 import werkzeug.utils
-from werkzeug.exceptions import BadRequest, Conflict, NotFound, UnprocessableEntity
+from werkzeug.exceptions import BadRequest, Conflict, NotFound, UnprocessableEntity, NotImplemented
```

Comment:
Since `NotImplemented` is used only in a single location, I would remove the import here and use `werkzeug.exceptions.NotImplemented` when raising it.